src/org/usfirst/frc4904/robot/Robot.java

```diff
@@ -55,16 +61,20 @@ public void alwaysExecute() {
 	}
 
 	@Override
-	public void disabledInitialize() {}
+	public void disabledInitialize() {
+	}
 
 	@Override
-	public void disabledExecute() {}
+	public void disabledExecute() {
+	}
 
 	@Override
-	public void testInitialize() {}
+	public void testInitialize() {
+	}
 
 	@Override
-	public void testExecute() {}
+	public void testExecute() {
+	}
```

Comment:
If these line breaks in between the {} are not a necessary consequence of the new formatter, please remove them as well.